libvast/vast/fwd.hpp

```diff
@@ -40,32 +174,41 @@ class Schema;
 
 #endif // VAST_ENABLE_ARROW
 
-namespace vast {
+// -- caf ----------------------------------------------------------------------
+
+// These are missing from <caf/fwd.hpp>.
 
-namespace fbs {
+namespace caf {
+
+template <class In>
+class inbound_stream_slot;
+
+} // namespace caf
+
+// -- vast::fbs ----------------------------------------------------------------
+
+namespace vast::fbs {
```

Comment:
Can change that back, I prefer this style because we need to open the `namespace vast` multiple times anyways because type IDs must be assigned in the global namespace.

Reply:
Done.